tern/analyze/common.py

```diff
@@ -445,3 +446,13 @@ def get_os_style(image_layer, binary):
                     package_manager=binary,
                     package_format=image_layer.pkg_format,
                     os_list=image_layer.os_guess), 'info'))
+
+
+def check_tar(tar_file):
+    '''Check if provided file is a valid tar archive file'''
+    try:
+        if tarfile.is_tarfile(tar_file):
+            return True
+    except FileNotFoundError:
+        pass
+    return False
```

Comment:
Can we not `return False` in the exception handling?

Reply:
I originally had the `return false` inside the exception handling, however, pylint complained:
```
pylint: inconsistent-return-statements / Either all return statements in a function should return an expression, or none of them should.
```

Reply:
Oh, it's because there is an `if` statement after the `try`. What about putting a `logger.error` statement instead of `pass`?

Reply:
Because there are a few places I use this check_tar() functionality I am hesitant to put a specific error message in the log when it is called. I was hoping to keep this function as a simple utility and we can handle the error message on a case by case basis depending on how/where in the code the function is invoked.

Reply:
What about:
```
if os.path.exists(tar_file):
    if tarfile.is_tarfile(tar_file):
        return True
return False
```